TACC/Core-CMS-Custom
# fix: wp-333 ecep search broken

## Overview

In ECEP, replace Portal search with Google search.

## Related

- [WP-333](https://jira.tacc.utexas.edu/browse/WP-333)
- requires https://github.com/TACC/Core-CMS/pull/743

## Changes

- **changed** CMS settings for search and portal
- **changed** CMS image to get search settings

## Testing

0. Use [TACC/Core-Portal-Deployments#30](https://github.com/TACC/Core-Portal-Deployments/pull/30) to deploy.
1. Open https://pprd.ecep.tacc.utexas.edu.
2. Verify search bar shows in header.
3. Verify portal nav does **not** show in header.
4. Use search engine to search for known text on site.
5. Verify search page loads with results.

## UI

![Screenshot 2023-11-07 at 2 37 08 PM](https://github.com/TACC/Core-CMS-Custom/assets/62723358/25549de2-bca9-47d9-9cb0-d0e6f51a38a0)

https://github.com/TACC/Core-CMS-Custom/assets/62723358/56b9977f-99fe-4b58-bdec-7521c533fc73
